### DISCUSSION TITLE
Auto breakup kick

### DISCUSSION THREAD

**Discussion by Zuwy101** [YimMenu/YimMenu#1063] [Ideas]
When I click "Block Join" on someone in player database, they can still join and also it just unchecks itself when i click the block join button. Im guessing this is because of one of the new gta updates.

So I ask please can you add this feature: Auto breakup kick,  a button you can put on someone on player database so that when they join you, you automatically breakup kick them without having to manually do it. This is good because it stops people coming to grief you or breakup kick you themselves when you don't expect it. Or could there be atleast a way to make it so that it alerts you in the top right if someone on your player database joins? If thats already a thing please tell me how.

**Comment by Yimura** [comment]
You need to click on save after setting the block join option.

**Reply by Zuwy101** [reply]
Oh ok thanks!
